### PR TITLE
homepage first draft

### LINKED ISSUE #11: simple django template for home page

### PR DESCRIPTION
**Pull request by annagav**
#### What are the relevant tickets?
Fix #11

### CODE REVIEW

File: static/css/layout.css [hunk 1]
```diff
@@ -0,0 +1,80 @@
+
```
Comment (noisecapella) [inline static/css/layout.css]: Can you copy this file's contents into `static/scss/layout.scss`? This wasn't working when you started with this functionality but it should work now

File: ui/templates/base.html [hunk 2]
```diff
@@ -3,6 +3,8 @@
   <head>
     <meta charset="UTF-8">
     <script type="text/javascript" src="{{ style_src }}"></script>
+    {% load staticfiles %}
+    <link href="{% static 'css/layout.css'%}" rel="Stylesheet"/>
```
Comment (noisecapella) [inline ui/templates/base.html]: After the contents of `layout.css` are copied into `layout.scss` you should remove this line. The contents of `layout.scss` are included in the above line with `style_src` instead.

File: ui/templates/index.html [hunk 3]
```diff
@@ -4,6 +4,38 @@
...
+            <div class="program_box">
+                <span style="line-height: 115px"></span>
+                <div class="program_name">{{ program }}</div>
+            </div>
+        {% endfor %}
+    </div>
+
+    <div class="footer_row">
+        <div class="footer_links">
+            <a href="#">MIT Giving</a>
```
Comment (noisecapella) [inline ui/templates/index.html]: Can you add URLs for these three links?

File: ui/urls.py [hunk 4]
```diff
@@ -7,4 +7,5 @@
 
 urlpatterns = [
     url(r'^$', index, name='ui-index'),
+
```
Comment (noisecapella) [inline ui/urls.py]: Remove this line please, since otherwise the file is unchanged

File: ui/templates/index.html (hunk 3 above)
Comment (annagav) [inline ui/templates/index.html]: MIT Giving : http://giving.mit.edu/ ?

File: static/scss/layout.scss [hunk 5]
```diff
@@ -1 +1,80 @@
-// Empty
+
```
Comment (noisecapella) [inline static/scss/layout.scss]: Can you change the underscores to dashes? It seems to be more popular to do it that way so we might as well follow along

File: static/scss/layout.scss [hunk 6]
```diff
@@ -1 +1,80 @@
...
+
+body{
```
Comment (noisecapella) [inline static/scss/layout.scss]: Can you add a space before the `{` for each item in this file?

File: ui/views.py [hunk 7]
```diff
@@ -4,6 +4,7 @@
 from django.conf import settings
 from django.contrib.staticfiles.templatetags.staticfiles import static
 from django.shortcuts import render
+from courses import models
```
Comment (noisecapella) [inline ui/views.py]: Can you change this to `from courses.models import Program`? It's nitpicky but it's a more specific import which is good for code cleanliness

**Comment by noisecapella** [comment]
It looks like you will need to edit the test to test for the existence of the program text instead of "Hi"

**Comment by noisecapella** [comment]
Also, feel free to provide a reason not to change something if you don't agree with a comment, instead of changing it, for any of our code reviews :)

File: ui/views_test.py [hunk 8]
```diff
@@ -17,10 +18,11 @@ def setUp(self):
 
     def test_index_view(self):
```
Comment (noisecapella) [inline ui/views_test.py]: Just two more things:
 - Can you rename this test to `test_program_liveness`
 - In this test create another Program where live=False, and assert that the program title is not in the response

File: ui/views_test.py [hunk 9]
```diff
@@ -17,10 +18,11 @@ def setUp(self):
...
         """Verify the index view is as expected"""
```
Comment (noisecapella) [inline ui/views_test.py]: Also can you update this description?

**Comment by noisecapella** [comment]
Looks great :+1: 

One more thing: Can you rename both the commit message and the PR title to something like "Implemented home page"? It should be in the past tense because our release script will use these titles as bullet points in our release notes

**Pull request merged**